packages/components/button/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@luniand-ui/button",
+    "version": "1.0.0",
+    "description": "Luniand UI | A component to display button in the browser component",
+    "keywords": [
+        "button"
+    ],
+    "author": "Albert Tran <trantoan.fox.97@gmail.com>",
```

Comment:
oh, not me, the author is your =))) please change your name and email. Same as with other component which you do

Reply:
it doesn't matter

Reply:
no, that is so important. It is ur effort. U should change to ur name